catalyst-gateway/bin/src/service/api/documents/mod.rs

```diff
@@ -78,10 +78,11 @@ impl DocumentApi {
         transform = "schema_version_validation"
     )]
     async fn put_document(
-        &self, /// The document to PUT
+        &self,
+        /// The document to PUT
         document: Cbor<Body>,
-        /// Authorization required.
-        _auth: CatalystRBACSecurityScheme,
+        // /// Authorization required.
+        // _auth: CatalystRBACSecurityScheme,
```

Comment:
Auth is required on PUT its only optional on GET or POST
Further the token use in the PUT must match one of the authors signing the document (if its multisig).
So, an actual author must PUT a document no one else can do it for them.

Reply:
yea yea, I know, I've just disabled it until @stanislav-tkach will fix this RBAC auth.